curriculum/challenges/english/01-responsive-web-design/css-grid/use-grid-area-without-creating-an-areas-template.english.md

```diff
@@ -34,7 +34,7 @@ Using the <code>grid-area</code> property, place the element with <code>item5</c
 
 ```yml
 tests:
-  - text: <code>item5</code> class should have a <code>grid-area</code> property such that it is between the third and fourth horizontal lines and between the first and fourth vertical lines.
+  - text: The <code>item5</code> class should have a <code>grid-area</code> to make it fill the whole area between the third and fourth horizontal lines, and first and fourth vertical lines.
```

Comment:
Minor suggestion, but since the challenge instructions refer to "`grid-area` property" - 

> Using the grid-area property,

It would be good to keep the usage of "property" in the new change as well - 

```suggestion
  - text: The <code>item5</code> class should have a <code>grid-area</code> property to make it fill the whole area between the third and fourth horizontal lines, and first and fourth vertical lines.
```

Reply:
Sounds good to me @Manish-Giri - want to commit that @AnonymousRepositoryStuff?